sdk/src/main/java/io/opentelemetry/sdk/metrics/MeterSdkRegistry.java

```diff
@@ -33,11 +33,11 @@
  * io.opentelemetry.OpenTelemetry}.
  */
 public final class MeterSdkRegistry implements MeterRegistry {
-  private final MeterSharedState sharedState;
```

Comment:
intentional?

Reply:
yes, this wasn't being used outside the constructor.

Reply:
yeah I figured, I just noticed it's not in line with `TracerSdkRegistry` but we can remove the property here until we actually need it